src/extras/BitmapText.js

```diff
@@ -469,10 +469,11 @@ export default class BitmapText extends core.Container
         const data = {};
         const info = xml.getElementsByTagName('info')[0];
         const common = xml.getElementsByTagName('common')[0];
+		const res = texture.baseTexture.resolution || 1;
```

Comment:
Small thing, could you change tabs to spaces here. 

Also, maybe this should default to `PIXI.settings.RESOLUTION` instead of `1`.... Like this: https://github.com/pixijs/pixi.js/blob/dev/src/core/textures/BaseTexture.js#L38